docs/scripts/generate_plugin_docs.py

```
@@ -961,13 +1003,15 @@ def generate_plugins_ecosystem_rst(self, all_plugins: List[Plugin]) -> str:
                 ]
                 if condition
             ]
-            tags_field = ",".join([category_tag] + extra_tags)
+            tags_field = ",".join(
+                filter(None, [category_tag] + plugin.tags + extra_tags)
```

Ruff (0.15.11)

[warning] Consider `[category_tag, *plugin.tags, *extra_tags]` instead of concatenation

Replace with `[category_tag, *plugin.tags, *extra_tags]`

(RUF005)